sway/commands/force_focus_wrapping.c

```diff
@@ -1,9 +1,15 @@
-#include <strings.h>
```

Comment:
Oops. Can you re-add these headers?

Reply:
Have I misunderstood something? They don't appear to be necessary here.

Reply:
Looks like `<strings.h>` isn't needed, but `sway/config.h` is needed for `config_add_swaynag_warning`

Reply:
I see, thanks for the explanation.